Emby.Server.Implementations/Services/ServiceController.cs

```diff
@@ -3,14 +3,27 @@
 using System.Threading.Tasks;
 using Emby.Server.Implementations.HttpServer;
 using MediaBrowser.Model.Services;
+using Microsoft.Extensions.Logging;
 
 namespace Emby.Server.Implementations.Services
 {
     public delegate object ActionInvokerFn(object intance, object request);
+
     public delegate void VoidActionInvokerFn(object intance, object request);
 
     public class ServiceController
     {
+        private readonly ILogger _log;
```

Comment:
Maybe rename to `_logger` to keep it consistent with the rest of the codebase?

Reply:
Done